lib/src/install.rs

```diff
@@ -1269,6 +1205,8 @@ async fn prepare_install(
     tracing::debug!("Target image reference: {target_imgref}");
 
     // A bit of basic global state setup
+    crate::mount::ensure_mirrored_host_mount("/dev")?;
+    crate::mount::ensure_mirrored_host_mount("/var/lib/containers")?;
```

Comment:
What if a user configures storage.conf to point to another path?

Reply:
Yeah, we already hit one instance of that. But fixing that is a distinct thing, orthogonal to dropping the requirements for the bind mounts being set up externally, right?

I will file a separate issue.